test/util/testUtils.js

```diff
@@ -0,0 +1,52 @@
+const mysql = require('mysql');
+const fs = require('fs');
+const path = require('path');
+
+const sqlFilePath = './dbReset.sql';
+const uploadsPath = '../../uploads';
+
+module.exports = {
+    resetDB: function () {
+        // reset database by sql script
+        fs.readFile(sqlFilePath, function (err, data) {
+            if (err) {
+                throw err;
+            }
+
+            const sqlQuery = data.toString();
+
+            let connection = mysql.createConnection({
+                host: 'localhost',
```

Comment:
Could you please set `host`, `user` and `password` as parameters since we have different credentials for our development instance and travis-ci. Thanks